CMGTools/TTHAnalysis/python/analyzers/ttHAlphaTVarAnalyzer.py

```diff
@@ -1,3 +1,6 @@
+from PhysicsTools.Heppy.analyzers.core.Analyzer import Analyzer
+from PhysicsTools.HeppyCore.utils.deltar import deltaR, deltaPhi
```

Comment:
yes you are right. I removed them (sorry I had changed those files before you did the migration to heppy and I did not notice the repetition)